### PR TITLE
Added notebook example of Amun Inspection Workflow

### PR DESCRIPTION
**Pull request by CermakM**
Signed-off-by: Marek Cermak <macermak@redhat.com>

new file:   notebooks/amun-inspection-workflow.ipynb

### CODE REVIEW

File: notebooks/amun-inspection-workflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,382 @@
+{
```
Comment (fridex) [inline notebooks/amun-inspection-workflow.ipynb:1]: Why are notebooks needed in this repo?

Comment (CermakM) [reply notebooks/amun-inspection-workflow.ipynb:1]: They aren't, I see them as examples of usage of the Argo Workflows.

Comment (CermakM) [reply notebooks/amun-inspection-workflow.ipynb:1]: I was thinking about whether they should rather be added to the `thoth-station/notebooks` repo, but I find it harder to link to when it comes to the outside contributors.

Comment (fridex) [reply notebooks/amun-inspection-workflow.ipynb:1]: Let's move them to `thoth-station/notebooks` - the repo is meant for notebooks and is automatically cloned to ODH JupyterHub instance. This way anyone can immediately try the notebooks. You can link to notebooks from the README file.

**Pull request closed without merging**